From 1bfe86d8e780f426889c9963436279b805d92e71 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Thu, 2 Jan 2025 18:23:42 +0100
Subject: [PATCH] dt-bindings: remoteproc: update mailbox properties in
 Cortex-M0 bindings

Remove mailbox rx and tx that are not used.

Fixes: 8962f14a3b73 ("dt-bindings: remoteproc: update Cortex-M0 bindings")

Change-Id: I82e89dc1bc43e8f175c4980bfa44f6f74c0f7ddd
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/425045
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 .../bindings/remoteproc/st,stm32-m0.yaml      | 21 ++-----------------
 1 file changed, 2 insertions(+), 19 deletions(-)

--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-m0.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-m0.yaml
@@ -40,29 +40,12 @@ properties:
 
   mboxes:
     description:
-      This property is required only if the rpmsg functionality is used.
-    items:
-      - description: |
-          A channel (a) used to communicate with the remote proc.
-          Bi-directional channel:
-            - from local to remote = send message
-            - from remote to local = send message ack
-      - description: |
-          A channel (b) working the opposite direction of channel (a)
-      - description: |
-          A channel (c) used by the local proc to notify the remote proc that it
-          is about to be shut down.
-          Unidirectional channel:
-            - from local to remote, where ACK from the remote means that it is
-              ready for shutdown
-    minItems: 1
+      A channel (c) used by the local proc to notify the remote proc that it
+      is about to be shut down.
 
   mbox-names:
     items:
-      - const: tx
-      - const: rx
       - const: shutdown
-    minItems: 1
 
   memory-region:
     description:
